www/docs/client/infer-types.md

```diff
@@ -55,7 +55,7 @@ Let's assume we have this example router:
 // @include: server
 ```
 
-Using the helpers, you can infer the types of the procedures. The following example shows how to infer the types of the procedures using the example `appRouter`:
+Using the helpers, you can infer the types of the procedures. The following example shows how to infer the types of the `create` procedure under the `post` namespace, using the example `appRouter`:
```

Comment:
Thanks for the suggestion, I agree it could benefit from a little revision, though I'm not sure it's quite there yet.

```suggestion
Using the helpers, we can infer the types of our router. The following example shows how to infer the types of the `post.create` procedure:
```

What do you think of this?

If you agree then please accept this suggestion or enable write access to your PR/fork so I can commit it. That way you retain your contributor credit! 🙂